drivers/sx127x/sx127x_getset.c

```diff
@@ -707,23 +707,13 @@ void sx127x_set_payload_length(sx127x_t *dev, uint8_t len)
     sx127x_reg_write(dev, SX127X_REG_LR_PAYLOADLENGTH, len);
 }
 
-static inline uint8_t sx127x_get_pa_select(uint32_t channel)
+static inline uint8_t sx127x_get_pa_select(const sx127x_t *dev)
```

Comment:
What about unify SX1272 and SX1276 in this function ?

Reply:
I'm not sure to understand your comment. This is already unified and more flexible I think: the new parameter `paselect` can be updated depending on the type of radio or how it is soldered on a breakout board.

Reply:
Oops sorry I didn't check the right version. I'll take another look.